app/komponenter/confirmationPanel.tsx

```diff
@@ -0,0 +1,38 @@
+import { ConfirmationPanel } from '@navikt/ds-react';
+import { useState } from 'react';
+import { Button } from '@navikt/ds-react';
+import { SanityDokument } from '~/typer/sanity/sanity';
+import TekstBlokk from './tekstBlokk/tekstBlokk';
+
+interface BekreftelsePanelProp {
+  melding: SanityDokument | undefined;
+  samtykke: SanityDokument | undefined;
+  feilmelding: SanityDokument | undefined;
+}
+
+const BekreftelsePanel: React.FC<BekreftelsePanelProp> = ({
+  melding,
+  samtykke,
+  feilmelding,
+}: BekreftelsePanelProp) => {
+  const [lest, settLest] = useState(false);
+  const [trykkVidere, settTrykkVidere] = useState(false);
+
+  return (
+    <div>
+      <ConfirmationPanel
+        checked={lest}
+        label={<TekstBlokk tekstblokk={samtykke} />}
+        onChange={() => settLest(x => !x)}
```

Comment:
Gjerne bruk variabelnavn som beskriver hva som settes. `x` kan være litt forvirrende om man ser raskt over

Reply:
Her er det faktisk en konvensjon om å kalle `x` for `prevState` i react :)